src/components/Body/HomeBody.css

```diff
@@ -0,0 +1,37 @@
+body {
```

Comment:
Tag need to be changed to specific element tag instead of the body. Image will be placed to every page that has a body.